backend/LexCore/Entities/User.cs

```diff
@@ -7,11 +7,18 @@ public class User : EntityBase
     public required bool IsAdmin { get; set; }
     public required string PasswordHash { get; set; }
     public required string Salt { get; set; }
+    public required bool EmailVerified { get; set; }
+    public required string? EmailVerificationToken { get; set; }
+    /// <summary>
+    /// The user's previous e-mail address
+    /// Only saved until the new address has been verified
+    /// </summary>
+    public string? PreviousEmail { get; set; }
```

Comment:
if you do end up keeping this I'd like it to have a name to indicate that it's temporary, otherwise it might seem like a bug that it gets set to null after verification when in reality there was a previous email.

Reply:
I think removing this is a good choice.